src/main/java/uk/gov/mca/beacons/service/model/Beacon.java

```diff
@@ -86,4 +99,12 @@ public LocalDate getLastServiced() {
   public void setLastServiced(LocalDate lastServiced) {
     this.lastServiced = lastServiced;
   }
+
+  public String getChkCode() {
+    return chkCode;
+  }
+
+  public void setChkCode(String chkCode) {
+    this.chkCode = chkCode;
+  }
```

Comment:
What is a `chkCode`?  Can't find reference to it on the MCGA's Sharepoint.  I've also left a comment on the wireframe:

![image](https://user-images.githubusercontent.com/54271433/107348276-b6e93780-6abe-11eb-8dd9-4a9e78e7eb48.png)

I think it refers to the [checksum number](https://beacons.org.nz/FAQs.aspx#What%20is%20Check%20Sum)?

Happy for this to be merged, just wanted to make sure I know the domain!